### ISSUE TITLE
Nullability mismatch on variables

### ISSUE THREAD

**Issue by grokys** [grokys/Octokit.GraphQL#66]
The following query doesn't work:

```
new Query()
    .Repository(Var("name"), Var("owner"))
    .Select(repository => repository.Name);
```

```
"Nullability mismatch on variable $name and argument owner (String / String!)"
"Nullability mismatch on variable $owner and argument name (String / String!)"
```

The variables created don't match the argument's nullability.

**Comment by grokys** [comment]
This is still happening:

```
var query = new Query()
    .Repository(Var("owner"), Var("name"))
    .PullRequest(Var("number"))
    .Select(x => x.Title)
    .Compile();
```

```
Nullability mismatch on variable $number and argument number (Int / Int!)
```

**Comment by achimkoellner** [comment]
Looks like $number is declared as non-required (Int) in the query signature although the schema expects it to be required (Int!)?

**Comment by grokys** [comment]
@achimkoellner exactly. I thought I'd fixed this in #67 but seems I didn't quite get it.

**Comment by grokys** [comment]
Actually looks like #67 did fix it.
